pkg/genericapiserver/server/config.go

```diff
@@ -43,17 +43,17 @@ import (
 	"k8s.io/apimachinery/pkg/version"
 	"k8s.io/apiserver/pkg/admission"
 	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/authentication/authenticatorfactory"
```

Comment:
> Anything that speak against k8s.io/apiserver/pkg/authentication/factory ?

You end up importing a package called `factory`.  I think it's the same reason for `net/http/httptest`